compiler/modules/modulegraphs.nim

```diff
@@ -617,3 +617,17 @@ proc onProcessing*(graph: ModuleGraph, fileIdx: FileIndex, moduleStatus: string,
         importStackLen: graph.importStack.len,
         moduleStatus: moduleStatus,
         fileIdx: fileIdx))
+
+proc isImportedException*(t: PType; conf: ConfigRef): bool =
+  # TODO: find a better home for this, it used to be `ast_query` but that
+  #       module must not depend upon `options` (due to `ConfigRef`). The
+  #       consumers of this proc are cgen, jsgen, semstmts, and tranf at the
+  #       time or writing.
```

Comment:
I think `compiler/ast/types.nim` would be a good fit.

Reply:
thanks, going to give that a shot now.